packages/api/tests/endpoints/token.js

```diff
@@ -204,6 +204,33 @@ test.serial('When using Earthdata Login, GET /token with a code stores the acces
   stub.restore();
 });
 
+test.serial('When OAuth provider returns no refresh token, GET /token with code and state results in redirect with access token', async (t) => {
```

Comment:
@yjpa7145 Now that I've added a test to prove that create an access token record succeeds with only an access token value, is this test necessary? The test is generally the same as other tests proving that the correct token is returned from a mocked access token response. The only difference here is explicitly not providing a `refreshToken` in the mocked access token response. But the only reason that missing property would matter is schema validation when creating the access token record and I added the other test to prove that it will not matter.

Reply:
I'm fine with removing this test.